migrations/db/migrations/20230607094822_add_auth_schema_protections.sql

```diff
@@ -87,7 +100,7 @@ create or replace function
     begin
       for cmd in select * from pg_event_trigger_dropped_objects()
       loop
-        if cmd.schema_name = 'auth' and cmd.object_identity not in ('auth.changes_seq', 'auth.lock_seq') then
+        if cmd.schema_name = 'auth' then
```

Comment:
Actually this is somewhat intentional as it appears that when you do a DROP the sequence/object is no longer visible when this event trigger runs. Essentially if we don't have that, we can never ever drop the sequences which is not a good idea.

Reply:
the concern is that dropping the sequences doesn't cascade to the event triggers which causes the event triggers to raise an error and prevent all migrations